docs/core/compatibility/sdk/6.0/version-file-entries.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Breaking change: .version file includes build version"
+description: Learn about a breaking change in the .NET SDK where the .version file now includes the specific build version.
+ms.date: 09/16/2022
+---
+# .version file includes build version
+
+The .NET SDK installation folder (under *dotnet\\sdk\<version>*) contains a *.version* file. This file includes the following data:
```

Comment:
```suggestion
The .NET SDK installation folder (under *dotnet/sdk/<version>*) contains a *.version* file. This file includes the following data:
```

The backslash is Windows, whereas the forward slash is both Unix and Windows. In other words, to be cross-platform, use the forward slash.